group_vars/matrix_servers

```diff
@@ -1329,6 +1329,34 @@ matrix_backup_borg_systemd_required_services_list: |
 # /matrix-backup-borg
 #
 ######################################################################
+######################################################################
+#
+# matrix-cactus-comments
+#
+######################################################################
+
+matrix_cactus_comments_enabled: false
+
+matrix_cactus_comments_container_image_self_build: "{{ matrix_architecture not in ['amd64', 'arm32', 'arm64'] }}"
+
+# Normally, matrix-nginx-proxy is enabled and nginx can reach cactus-comments over the container network.
+# If matrix-nginx-proxy is not enabled, or you otherwise have a need for it, you can expose
+# cactus-comments web-server port.
+#matrix_cactus_comments_container_http_host_bind_port: "{{ '' if matrix_nginx_proxy_enabled else '127.0.0.1:' + matrix_cactus_comments_container_port | string }}"
+
+matrix_cactus_comments_systemd_required_services_list: |
+  {{
+    (['docker.service'])
+    +
+    (['matrix-' + matrix_homeserver_implementation + '.service'])
+  }}
+matrix_cactus_comments_homeserver_url: "{{ matrix_homeserver_url }}"
```

Comment:
Can this be changed to `matrix_homeserver_container_url` (which is usually someting lke `http://matrix-nginx-proxy:12080` if nginx-proxy is enabled and something like `http://matrix-synapse:8008` otherwise) or does Cactus Comments expect a public URL (e.g. `https://matrix.DOMAIN`)?

https://cactus.chat/docs/server/self-host/ seems to indicate that container URLs also work.

I don't see a separate configuration option for configuring the public homeserver URL though.. I wonder how it auto-detects it, etc., for generating the URLs for the client embed files.

Reply:
Yep, tested it and works 🥳